css-brochure-form/index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Challenge - Brochure Form</title>
+    <link rel="stylesheet" href="fonts.css" />
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <h1>LEARN MORE</h1>
+    <form>
+      <div>
+        <label for="email" class="company l1"><b>COMPANY EMAIL</b></label>
```

Comment:
If you are using `b` just for making an element bold, then I would get rid of it. You want to apply your styling via css. So I would either change the `b` to a `strong` to make it a better semantic element, or I would adjust the `font-weight` of the `label` to just make the text look bold